BlueprintUI/Sources/Environment/Environment.swift

```diff
@@ -57,4 +59,9 @@ public struct Environment {
             values[ObjectIdentifier(key)] = newValue
         }
     }
+    
+    /// Cache used to speed up measurements during a layout pass.
+    /// This is declared on the `Environment` directly, because access to the `measurementCache` is a hot path
+    /// which we can optimize by not needing the usual `env.values[...]` lookup.
```

Comment:
Surprised to see this. How slow is it? As a dictionary lookup I would expect it to be quite fast.

Reply:
It wasn't a lot, but it was in the hot path, so it was on the Instruments trace